gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html

```diff
@@ -413,6 +413,15 @@ <h3 class="portal__form__card__title">Documentation</h3>
           <mat-label>Documentation URL</mat-label>
           <input formControlName="url" matInput type="text" />
         </mat-form-field>
+        <mat-form-field
+          matTooltip="Page not found message"
```

Comment:
What do you think about getting a bit more specific on the tooltip?

```suggestion
          matTooltip="Message to be displayed when a Documentation Page is not found"
```